scriptworker/poll.py

```diff
@@ -121,15 +127,33 @@ def get_azure_urls(context):
 
     Returns:
         dict: the claimTask json
+
+    Raises:
+        ScriptWorkerException: on failure of request_function
```

Comment:
hm, not sure if this is true. see below. Also, in the docs above the:

    "Delete the message from the Azure queue whether the claim was successful or not (error 409 on claim means the task was cancelled/expired/claimed)."

seems to be outdated now since we don't try to delete if we get a non 409 exception.